index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
```

Comment:
You might use a templating engine (haml / erb) to clean this up a bit. Not sure how that would play with jasmine-jquery though... you would need to precompile. So maybe vanilla HTML is better actually. :+1: